frontend/src/components/common/AddAdminModal.tsx

```diff
@@ -0,0 +1,189 @@
+import React, { useState } from "react";
+import {
+  Button,
+  FormControl,
+  FormLabel,
+  Input,
+  Modal,
+  ModalHeader,
+  ModalCloseButton,
+  ModalContent,
+  ModalBody,
+  ModalFooter,
+  ModalOverlay,
+  Radio,
+  RadioGroup,
+  HStack,
+  Text,
+  useDisclosure,
+} from "@chakra-ui/react";
+import { PlusOutlineIcon } from "./icons";
+
+const AddAdminFormLabel = ({ text }: { text: string }): React.ReactElement => {
+  return (
+    <FormLabel
+      as="legend"
+      color="blue.300"
+      mb={0}
+      fontSize="20px"
+      lineHeight="26px"
+    >
+      {text}
+    </FormLabel>
+  );
+};
+
+const AddAdminTextInput = ({
```

Comment:
Maybe we can actually refactor `AddAdminTextInput` and `AddAdminFormLabel` to be a general components for our codebase when creating forms? Let me know if there's some reason why this shouldn't be the case!